### DISCUSSION TITLE
Unable to connect to relay

### DISCUSSION THREAD

**Discussion by TTF-fog** [PrismarineJS/bedrock-protocol#349] [Q&A]
trying to connect to a relay causes my game to freeze on "connecting to external server". According to the people on the discord, it's supposed to kick you and ask you to sign in, but nothing like that happens for me, and using prismarine-auth does not yield any results.
I'm using the same code as createRelay.js in examples

**Comment by extremeheat** [comment]
This is likely a firewall or loop back issue, if it's a problem with the Win10 edition. See the FAQ.

**Comment by TTF-fog** [comment]
I've done the loopback exemption already, and my firewall is disabled but the same thing happens

**Comment by extremeheat** [comment]
Host the server on a different device or use a cloud service

**Comment by TTF-fog** [comment]
I used gitpod, but the relay refused to show up

**Comment by extremeheat** [comment]
Then you have a firewall issue. If you can't connect to a vanilla server + client with this setup, then bedrock-protocol will not work either.

**Comment by TTF-fog** [comment]
with what setup?

**Comment by extremeheat** [comment]
Since this is not an issue with bedrock-protocol, I’ve moved the issue to a discussion if others would like to help.
